src/app/admin/booth/add-booth/add-booth.component.html

```diff
@@ -16,8 +16,12 @@
         Mængde af stande:
       </label>
       <input class="form-control" type="number" placeholder="Mængde af stande" formControlName="amount">
+      <small class="form-text text-muted">
+        Et maximum af {{maxAmountOfBoothsAllowed}} stande kan blive lavet ad gangen.
+      </small>
     </div>
 
     <button class="btn btn-primary" type="submit">Tilføj stande</button>
   </form>
+
 </div>
```

Comment:
Missing:
<div *ngFor="let alert of alerts">
    <alert [type]="alert.type" [dismissOnTimeout]="alert.timeout" (onClosed)="onAlertClosed(alert)" [dismissible]="true"><strong>{{ alert.msgStrong }}</strong> {{ alert.msg }}</alert>
  </div>